xchange-stream-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingService.java

```diff
@@ -14,6 +15,16 @@ public BinanceStreamingService(String baseUri, ProductSubscription productSubscr
     this.productSubscription = productSubscription;
   }
 
+  public BinanceStreamingService(String baseUri,
```

Comment:
You've added these constructor overloads but no code that uses them. Are you planning to call them on construction in the `StreamingExchange` implementations in some follow-up PR?

Reply:
I don't have immediate plans for them, I think being restricted to use the defaults is not ideal and I use these outside of this codebase (I think it makes sense to have them in place)

Reply:
I'd be intrigued to hear how and why you are using the underlying streaming services directly (rather than just exposing nice entry points in the `StreamingExchange`s, but fair enough for now.

Reply:
@badgerwithagun in short I want raw json payloads, I'm not interested in the unified objects Xchange uses so I have found a few ways of accomplishing this and using the mature parts of Xchange's code.

Reply:
thanks for approving